apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/trace/SpanLayer.java

```diff
@@ -26,7 +26,8 @@
     RPC_FRAMEWORK(2),
     HTTP(3),
     MQ(4),
-    CACHE(5);
+    CACHE(5),
+    COROUTINE(6);
```

Comment:
This should not be added. `COROUTINE` is a local span. Should not be `Layer`.

Reply:
Got it, I create it for other incoming coroutine plugins, like goroutine etc.